Makefile.generated_files

```diff
@@ -12,8 +12,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
```

Comment:
Since we are calling this using an explicit `$(MAKE) -f Makefile.$@ $@`, we should also set the SHELL here:
```
SHELL := /bin/bash
```
I had to do this to get this to get `make generated_files` to work